Gems/WarehouseAutomation/Code/Include/WarehouseAutomation/WarehouseAutomationBus.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <WarehouseAutomation/WarehouseAutomationTypeIds.h>
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace WarehouseAutomation
+{
+    class WarehouseAutomationRequests
+    {
+    public:
+        AZ_RTTI(WarehouseAutomationRequests, WarehouseAutomationRequestsTypeId);
+        virtual ~WarehouseAutomationRequests() = default;
+        // Put your public methods here
```

Comment:
There seems to be no methods. Are there any planned or should we discard the interface?

Reply:
I don't really know the planned expansion direction for this gem so I decided to leave such files in (they're much easier to delete later than to add from scratch). I imagine there is some likelihood of "global methods" being needed in the future.
Do you think such methods are unlikely? If so, I'll delete this interface, and probably also this gem's SystemComponents.

Reply:
This Gem is a "bunch of things" rather than a system, so yes I think the interface would not be needed.